### PR TITLE
Update agenda.lib.php

### PR DESCRIPTION
**Pull request by jmontoyaa**

### CODE REVIEW

File: main/calendar/agenda.lib.php [hunk 1]
```diff
@@ -1336,10 +1336,16 @@ public function get_course_events($start, $end, $courseInfo, $groupId = 0, $sess
 
         $dateCondition = null;
         if (!empty($start)  && !empty($end)) {
-            $dateCondition .= "AND (
-                (agenda.start_date >= '".$start."' OR agenda.start_date IS NULL) AND
-                (agenda.end_date <= '".$end."' OR agenda.end_date IS NULL)
-            )";
+             $dateCondition .= "AND (
+                 agenda.start_date BETWEEN '".$start."' AND '".$end."' OR
+                 agenda.end_date BETWEEN '".$start."' AND '".$end."'
```
Comment (ywarnier) [inline main/calendar/agenda.lib.php]: Missing an AND or OR at the end of this line, I believe...

**Comment by ywarnier** [comment]
Not sure what this fixes or why you left it as PR, but I added a note just in case (missing a conditional statement)

**Comment by jmontoyaa** [comment]
Thanks for the revision even if I didn't create the PR :)

**Comment by jmontoyaa** [comment]
Mmm I think I did create the PR while creating the patch.

**Pull request merged**